pkgs/applications/audio/furnace/default.nix

```diff
@@ -0,0 +1,96 @@
+{ stdenv
+, lib
+, nix-update-script
+, fetchFromGitHub
+, fetchpatch
+, cmake
+, pkg-config
+, makeWrapper
+, fmt_8
+, libsndfile
+, SDL2
+, zlib
+, withJACK ? stdenv.hostPlatform.isUnix
+, libjack2
+, withGUI ? true
+, Cocoa
+, libobjc
+}:
+
+let
+  version = "0.5.5";
+  src = fetchFromGitHub {
+    owner = "tildearrow";
+    repo = "furnace";
+    rev = "v${version}";
+    fetchSubmodules = true;
+    sha256 = "sha256-8CHRrfCe/lLPH6LYpzJUqXPSAe83v0zU18ooIDLtF84=";
+  };
+  # SDL 2.0.18+ is needed, not packages yet
+  # https://github.com/NixOS/nixpkgs/issues/154137
```

Comment:
I think we can wait for https://github.com/NixOS/nixpkgs/pull/136166 which should be merged in the next days.

Reply:
Targeting staging with merged SDL2 bump now.